src/collections/_documentation/development/sdk-dev/overview.md

```diff
@@ -95,13 +95,13 @@ a basic message or exception:
 SDKs are encouraged to allow arbitrary options via the constructor, but must allow the first argument as a DSN string. This string contains the following bits:
 
 ```
-'{PROTOCOL}://{PUBLIC_KEY}:{SECRET_KEY}@{HOST}/{PROJECT_ID}'
+'{PROTOCOL}://{PUBLIC_KEY}:{SECRET_KEY}@{HOST}/{PATH}/{PROJECT_ID}'
 ```
 
 The final endpoint you’ll be sending requests to is constructed per the following:
 
 ```
-'{URI}/api/{PROJECT_ID}/store/'
+'{PROTOCOL}://{HOST}/{PATH}/api/{PROJECT_ID}/store/'
```

Comment:
I would write this differently.
Something like `{URI} = {PROTOCOL}://{HOST}/{PATH}` where `/{PATH}` is optional.
And keep the old `'{URI}/api/{PROJECT_ID}/store/'`

Reply:
done